contracts/pm/mixins/MixinTicketProcessor.sol

```diff
@@ -13,7 +13,7 @@ contract MixinTicketProcessor is MContractRegistry, MTicketProcessor {
     uint256 public ticketValidityPeriod;
 
     /**
-     * @dev Process sent funds.
+     * @notice Process sent funds.
```

Comment:
All of the functions in this file are internal and are never used by an external account so it feels like `@dev` should be used instead.

Reply:
fixed in b79bb67